components/connection/WellKnownURLs.tsx

```diff
@@ -8,61 +8,86 @@ const links = [
     description:
       'The metadata file that your customers who use federated management systems like OpenAthens and Shibboleth will need to configure your service.',
```

Comment:
@deepakprabhakara Should we move these to the `/locales/en/common.json`?

Reply:
Definitely

Reply:
@devkiran done